### ISSUE TITLE
Pre-work Checklist: Developer: Christopher Langley

### ISSUE THREAD

**Issue by Chrisklangley** [hackforla/website#5729]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by roslynwythe** [comment]
Hi @Chrisklangley please add the issue to the `Project Board`.  You can find the `Projects` setting under the `Labels` setting on the right column of the issue.  When you have done that, please add the `ready for dev lead` label just so it will come to my attention so I can approve it.    Thanks!

**Comment by Chrisklangley** [comment]
Availability: Tuesday 6pm-9pm
Wednesday: 6pm-9pm
Thursday 6pm-9m 
Sunday all day

**Comment by Chrisklangley** [comment]
ETA for issue: Wednesday 9am

**Comment by HackforLABot** [comment]
## Activity Log: Chrisklangley
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- Chrisklangley opened: https://github.com/hackforla/website/issues/5729 at 10/16/2023, 08:38 PM PDT
- Chrisklangley assigned: https://github.com/hackforla/website/issues/5729 at 10/16/2023, 08:38 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/issues/5729#issuecomment-1765621863 at 10/16/2023, 08:54 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/issues/5729#issuecomment-1765622641 at 10/16/2023, 08:55 PM PDT
- Chrisklangley assigned: https://github.com/hackforla/website/issues/5620 at 10/19/2023, 07:47 PM PDT
- Chrisklangley closed as completed: https://github.com/hackforla/website/issues/5729#event-10719236743 at 10/19/2023, 08:18 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/issues/5620#issuecomment-1773435053 at 10/20/2023, 02:48 PM PDT
- Chrisklangley opened: https://github.com/hackforla/website/pull/5745 at 10/20/2023, 02:50 PM PDT
- Chrisklangley closed: https://github.com/hackforla/website/pull/5745#event-10733849601 at 10/22/2023, 12:12 PM PDT
- Chrisklangley opened: https://github.com/hackforla/website/pull/5765 at 10/22/2023, 12:39 PM PDT
- Chrisklangley closed by PR 5765: https://github.com/hackforla/website/issues/5620#event-10734836262 at 10/22/2023, 06:53 PM PDT
- Chrisklangley PR was merged: https://github.com/hackforla/website/pull/5765#event-10735298724 at 10/22/2023, 08:27 PM PDT
- Chrisklangley issue closed by PR 5765: https://github.com/hackforla/website/issues/5620#event-10735298807 at 10/22/2023, 08:27 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5771#issuecomment-1780259691 at 10/25/2023, 06:04 PM PDT
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5771#pullrequestreview-1698576622 at 10/25/2023, 06:18 PM PDT
- Chrisklangley assigned: https://github.com/hackforla/website/issues/5759 at 10/25/2023, 06:29 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/issues/5759#issuecomment-1780277687 at 10/25/2023, 06:31 PM PDT
- Chrisklangley opened: https://github.com/hackforla/website/pull/5789 at 10/27/2023, 02:37 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5785#issuecomment-1783549638 at 10/27/2023, 02:51 PM PDT
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5785#pullrequestreview-1702617345 at 10/27/2023, 03:34 PM PDT
- Chrisklangley PR was merged: https://github.com/hackforla/website/pull/5789#event-10800309855 at 10/29/2023, 10:50 AM PDT
- Chrisklangley issue closed by PR 5789: https://github.com/hackforla/website/issues/5759#event-10800309891 at 10/29/2023, 10:50 AM PDT
- Chrisklangley assigned: https://github.com/hackforla/website/issues/5854 at 11/08/2023, 05:34 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/issues/5854#issuecomment-1804387064 at 11/09/2023, 10:48 AM PST
- Chrisklangley opened: https://github.com/hackforla/website/pull/5889 at 11/09/2023, 12:07 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5907#issuecomment-1813133984 at 11/15/2023, 11:29 AM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5907#pullrequestreview-1732844552 at 11/15/2023, 11:57 AM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5908#issuecomment-1813761434 at 11/15/2023, 08:11 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5917#issuecomment-1817314757 at 11/17/2023, 05:41 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5917#pullrequestreview-1738164083 at 11/17/2023, 06:17 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5908#pullrequestreview-1738518882 at 11/18/2023, 04:59 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5889#issuecomment-1817949328 at 11/19/2023, 11:11 AM PST
- Chrisklangley PR was merged: https://github.com/hackforla/website/pull/5889#event-11004706677 at 11/19/2023, 02:05 PM PST
- Chrisklangley issue closed by PR 5889: https://github.com/hackforla/website/issues/5854#event-11004706707 at 11/19/2023, 02:05 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5918#issuecomment-1822005763 at 11/21/2023, 06:49 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/5914#issuecomment-1822006054 at 11/21/2023, 06:49 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5914#pullrequestreview-1743492115 at 11/21/2023, 07:34 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5918#pullrequestreview-1743518178 at 11/21/2023, 07:51 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5928#pullrequestreview-1748388895 at 11/24/2023, 11:56 AM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/5914#pullrequestreview-1749451953 at 11/26/2023, 03:46 PM PST
- Chrisklangley assigned: https://github.com/hackforla/website/issues/5831 at 11/26/2023, 04:18 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/issues/5831#issuecomment-1826958780 at 11/26/2023, 04:26 PM PST
- Chrisklangley opened: https://github.com/hackforla/website/pull/6007 at 12/14/2023, 02:37 PM PST
- Chrisklangley PR was closed: https://github.com/hackforla/website/pull/6007#event-11273492093 at 12/18/2023, 12:53 AM PST
- Chrisklangley issue closed as not planned: https://github.com/hackforla/website/issues/5831#event-11273522579 at 12/18/2023, 12:56 AM PST
- Chrisklangley assigned: https://github.com/hackforla/website/issues/5984 at 12/20/2023, 05:27 PM PST
- Chrisklangley unassigned: https://github.com/hackforla/website/issues/5984 at 01/03/2024, 04:46 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6056#pullrequestreview-1803328366 at 01/03/2024, 05:59 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6100#issuecomment-1889991685 at 01/12/2024, 01:47 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6100#pullrequestreview-1823989495 at 01/16/2024, 08:15 AM PST
- Chrisklangley assigned: https://github.com/hackforla/website/issues/6137 at 01/26/2024, 05:49 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/issues/6137#issuecomment-1912909870 at 01/26/2024, 05:53 PM PST
- Chrisklangley opened: https://github.com/hackforla/website/pull/6194 at 01/29/2024, 01:21 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6190#issuecomment-1915695045 at 01/29/2024, 02:35 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6190#pullrequestreview-1849918089 at 01/29/2024, 02:48 PM PST
- Chrisklangley assigned: https://github.com/hackforla/website/issues/1976#issuecomment-1848917244 at 01/30/2024, 01:03 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6198#issuecomment-1918299040 at 01/30/2024, 07:04 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6197#issuecomment-1918300729 at 01/30/2024, 07:06 PM PST
- Chrisklangley PR was merged: https://github.com/hackforla/website/pull/6194#event-11650255302 at 01/30/2024, 07:18 PM PST
- Chrisklangley issue closed by PR 6194: https://github.com/hackforla/website/issues/6137#event-11650255429 at 01/30/2024, 07:18 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6197#pullrequestreview-1859789849 at 02/02/2024, 09:05 AM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6262#issuecomment-1933292567 at 02/07/2024, 07:06 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6262#pullrequestreview-1869228154 at 02/07/2024, 07:56 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6260#issuecomment-1933337383 at 02/07/2024, 08:15 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6260#pullrequestreview-1869244172 at 02/07/2024, 08:18 PM PST
- Chrisklangley unassigned: https://github.com/hackforla/website/issues/1976#issuecomment-1917889892 at 02/08/2024, 03:53 PM PST
- Chrisklangley assigned: https://github.com/hackforla/website/issues/4756 at 02/08/2024, 04:25 PM PST
- Chrisklangley unassigned: https://github.com/hackforla/website/issues/4756 at 02/08/2024, 04:25 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6267#issuecomment-1935146008 at 02/08/2024, 04:33 PM PST
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6283#pullrequestreview-1879235817 at 02/13/2024, 05:46 PM PST
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6678#issuecomment-2062801232 at 04/17/2024, 06:00 PM PDT
- Chrisklangley commented on: https://github.com/hackforla/website/pull/6821#issuecomment-2095136514 at 05/05/2024, 08:37 PM PDT
- Chrisklangley submitted review: https://github.com/hackforla/website/pull/6821#pullrequestreview-2039916247 at 05/05/2024, 08:39 PM PDT
